src/libs/ReportActionsUtils.ts

```diff
@@ -806,6 +806,29 @@ function hasRequestFromCurrentAccount(reportID: string, currentAccountID: number
     return reportActions.some((action) => action.actionName === CONST.REPORT.ACTIONS.TYPE.IOU && action.actorAccountID === currentAccountID);
 }
 
+function isReportActionUnread(reportAction: OnyxEntry<ReportAction>, lastReadTime: string) {
+    if (!lastReadTime) {
+        return Boolean(!isCreatedAction(reportAction));
+    }
+
+    return Boolean(reportAction && lastReadTime && reportAction.created && lastReadTime < reportAction.created);
+}
+
+/**
+ * Check whether the report action of the report is unread or not
```

Comment:
It's a strange way to phrase it because "current" can be interpreted in different ways. Better to say - "the report action passed to this method" and not "the current report action of the report".